MediaAdminBundle/Tests/EventSubscriber/MediaDeletedSubscriberTest.php

```diff
@@ -15,18 +15,24 @@ class MediaDeletedSubscriberTest extends \PHPUnit_Framework_TestCase
     protected $subscriber;
 
     protected $fileAlternativesManager;
-    protected $event;
-    protected $media;
+    protected $media1;
+    protected $media2;
+    protected $event1;
+    protected $event2;
 
     /**
      * Set Up the test
      */
     public function setUp()
     {
-        $this->media = Phake::mock('OpenOrchestra\Media\Model\MediaInterface');
+        $this->media1 = Phake::mock('OpenOrchestra\Media\Model\MediaInterface');
+        $this->media2 = Phake::mock('OpenOrchestra\Media\Model\MediaInterface');
 
-        $this->event = Phake::mock('OpenOrchestra\MediaAdmin\Event\MediaEvent');
-        Phake::when($this->event)->getMedia()->thenReturn($this->media);
+        $this->event1 = Phake::mock('OpenOrchestra\MediaAdmin\Event\MediaEvent');
+        Phake::when($this->event1)->getMedia()->thenReturn($this->media1);
+
+        $this->event2 = Phake::mock('OpenOrchestra\MediaAdmin\Event\MediaEvent');
+        Phake::when($this->event2)->getMedia()->thenReturn($this->media2);
 
         $this->fileAlternativesManager = Phake::mock('OpenOrchestra\MediaAdmin\FileAlternatives\FileAlternativesManager');
         $this->subscriber = new MediaDeletedSubscriber($this->fileAlternativesManager);
```

Comment:
@nicolasThal : I don't understand your comment. Where do you think there's a mistake with this interface ?